### PR TITLE
Validate coerced properties with biz-ops-schema

### PR DESCRIPTION
**Pull request by chee**
Ah, so after doing the work to make it run the `validateProperty` on fields after they’ve been produced by the coercer, i went to write a test for it.
The only thing that uses a string pattern in `System` so far is `Code` (which we aren’t going to allow them to set in the runbook, haha).
I’ve pushed the code up anyway, because it will work once we add something like a url regex to the validation strings.

### CODE REVIEW

File: src/lib/runbook.md.js [hunk 1]
```diff
@@ -21,11 +19,12 @@ async function runbookMd() {
 
 	const typeNames = new Set(types.map(type => type.name));
 
+	const validateProperty = (key, value) => {
+		return schema.validateProperty('System', key, value)
```
Comment (doramatadora) [inline src/lib/runbook.md.js]: Prettier will complain about a missing semicolon here and on the line below

File: src/lib/runbook.md.js [hunk 2]
```diff
@@ -38,6 +37,9 @@ async function runbookMd() {
 			primitiveTypesMap: schema.primitiveTypesMap,
 			enums: schema.getEnums(),
 		})
+		.use(validateBizopsProperties, {
+			validateProperty
```
Comment (doramatadora) [inline src/lib/runbook.md.js]: Prettier will complain about a missing comma here

**Comment by doramatadora** [comment]
Looks great 👍 Requested changes are linting.

File: src/lib/runbook.md.js (hunk 1 above)
Comment (chee) [reply src/lib/runbook.md.js]: ah, thanks 😊 i'll do a little rebase

**Comment by chee** [comment]
> Looks great 👍 Requested changes are linting.

Thanks !! i did a little history rewriting and now the linting has always have been right 😊

**Pull request merged**